test/functional/test_framework/test_framework.py

```diff
@@ -213,7 +214,7 @@ def run_test(self):
 
     # Public helper methods. These can be accessed by the subclass test scripts.
 
-    def add_nodes(self, num_nodes, extra_args=None, rpchost=None, timewait=None, binary=None):
+    def add_nodes(self, num_nodes, extra_args=None, rpchost=None, timewait=None, binary=None, bind=True):
```

Comment:
Nit, but should this argument also be `bind_to_localhost_only`? 

`bind` is a bit ambiguous.

Reply:
How is this argument passed along in this function?

Reply:
it isn't used